src/main/java/com/redhat/cloud/policies/app/rest/utils/PagingUtils.java

```diff
@@ -118,10 +118,22 @@ public static Pager extractPager(@NotNull UriInfo uriInfo) {
             if (filterMatcher.find()) {
                 String column = filterMatcher.group(1);
                 String value = queryParams.getFirst(key);
-                String operatorString = queryParams.getFirst(String.format("%s[%s]", FILTER_OP, column));
-                Filter.Operator operator = Filter.Operator.EQUAL;
-                if (operatorString != null) {
-                    operator = Filter.Operator.fromName(operatorString);
+                Filter.Operator operator;
+                if (column.equals("is_enabled")) {
```

Comment:
The only "problem" is that `PagingUtils` was meant to be general across all our resources. 
I think we will figure out what to do IF we want to have more resources filtered, not an issue right now.

Reply:
Yes. I had the same thoughts. We would need some datastructure (additional fields on the `Filter.Operator` enum) to list applicable fields or vice versa.
But yes, let's think about it when we need it.